Improves Blockchain Basics (#1165)
maticnetwork/matic-docs: docs/home/blockchain-basics/import-account-to-metamask.md

---
import useBaseUrl from '@docusaurus/useBaseUrl';

> You can always import your account to Metamask if you're using any other web/mobile wallet. All you need to to make sure that you have a backup of your seed phrase.

Reviewer: ```suggestion
You can always import your account to Metamask if you're using any other web / mobile wallet. All you need is to make sure that you have a backup of your seed phrase.
```

### Importing your account to Metamask

Keep your seed phrase handy now.

> When you click on Import Wallet, you will be navigated to a different screen which will ask you to enter your seedphrase. You will also be asked to set a password. 

Reviewer: ```suggestion
When you click on Import Wallet, you will be navigated to a different screen which will ask you to enter your seed phrase. You will also be asked to set a password. 
```

By default, only ETH is listed in Metamask. If you already have Matic tokens then that too will be listed automatically. In order to view your other tokens, you can click on Add Tokens and search the token you want to be listed.

> Do keep extreme care of your seed phrase and keep backups, because unlike exchanges, you need to keep your seed phrase safe. If the seed phrase is lost, funds are not recoverable. Also never share this seed phrase with anyone.

Reviewer: ```suggestion
Do keep extreme care of your seed phrase and keep backups, because unlike exchanges, you need to keep your seed phrase safe. If the seed phrase is lost, funds are not recoverable. Also, never share this seed phrase with anyone.
```
